src/Equinor.ProCoSys.IPO.Command/InvitationCommands/CreateInvitation/CreateInvitationCommandHandler.cs

```diff
@@ -348,6 +348,11 @@ private async Task<List<CommPkg>> GetCommPkgsToAddAsync(IList<string> commPkgSco
                 throw new IpoValidationException("Could not find all comm pkgs in scope.");
             }
 
+            if (commPkgDetailsList.Any(c => c.OperationHandoverStatus == "Accepted"))
```

Comment:
When end user choose CommPkgs from UI and make commPkgScope,
![image](https://github.com/equinor/procosys-invitation-for-punch-out-api/assets/19545528/d5d8ce1b-70c6-4317-9c86-25302975286e)
...can user choose among CommPkgs with .OperationHandoverStatus == "Accepted", or are they filtered to be not able to choose?

If user can choose such, throwing exception will be wrong

Reply:
Ahh .. I now see that you throw IpoValidationException, which is some kind of correct.
However, the end user still don't know which CommPkgs cause the problem for him/her.

We should discuss what to do ...
Possible solution: 
1) Automatically filter out CommPkgs with OperationHandoverStatus == "Accepted" in endpoint so end user can't choose them (Then it is ok with IpoValidationException without more details because end user should not get there)
2) In UI in IPO, visualize CommPkgs with OperationHandoverStatus == "Accepted" somehow, and prohibit end user to choose them (Then it is ok with IpoValidationException without more details because end user should not get there)
3) If it is very unlikely that end user erroneously choose among CommPkgs with OperationHandoverStatus == "Accepted", we let solution be as is, but show more details about CommPkgs with OperationHandoverStatus == "Accepted" in thrown exception so end user can correct by unselecting those CommPkgs

Reply:
There will be a UI case to use the new property, OperationHandoverStatus, in frontend to disable commpkgs that have RFOC accepted (same with MC pkgs). So from UI it should not be possible to select these, but I can still add the packages in the exception message

Reply:
Approved